Baby Monitor/Source Files/Services/Connection/SocketCommunicationManager.swift

```diff
@@ -41,3 +50,14 @@ class DefaultSocketCommunicationManager: SocketCommunicationManager {
         communicationTerminationPublisher.onNext(())
     }
 }
+
+private extension DefaultSocketCommunicationManager {
```

Comment:
Let's not make a private extension and keep the project convention.

Reply:
👍